src/main/java/com/j256/ormlite/misc/functional/Supplier.java

```diff
@@ -0,0 +1,14 @@
+package com.j256.ormlite.misc.functional;
+
+/**
+ * This is a functional interface intended to support lambda expressions in ormlite.
+ */
+public interface Supplier<T> {
```

Comment:
Maybe `EntityCreator` would be a better name?  `Supplier` is not descriptive enough.

Reply:
This is the common name for this kind of functional interface. I'm not sure. It will probably be easier to use names familiar to developers for development